src/repositories/tweet.repository.ts

```diff
@@ -0,0 +1,37 @@
+import { ObjectId } from "mongodb";
+
+import { TweetModel } from "../models/Tweet";
+
+export const getOneById = (id: ObjectId) => {
+  return TweetModel.findById(id);
+};
+export const createOne = async (
```

Comment:
no reason to make a function async without await in the body
same comment for the rest functions

https://palantir.github.io/tslint/rules/no-async-without-await/#:~:text=Marking%20a%20function%20as%20async,keyword%20is%20there%20by%20mistake.